andela/activo-web
# #159528142 Fix display add assetpane bug

**What does this PR do?**
Fixes the add asset-pane display bug

**Description of Task to be completed?**
- change the value of measuring height from vh to percent in addassetpane.scss file

**How should this be manually tested?**
After cloning, start the application and open up your browser. Open up your console and scroll up/down. The add asset pane should not shrink, and neither should the form.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#159528142](https://www.pivotaltracker.com/story/show/159528142)

**Screenshots (if appropriate)**
<img width="1440" alt="screen shot 2018-08-04 at 4 11 26 pm" src="https://user-images.githubusercontent.com/24798364/43677869-376213d4-9801-11e8-96f5-8d9f5efb00d4.png">
-------------------------------------------------------------
<img width="1440" alt="screen shot 2018-08-04 at 4 11 17 pm" src="https://user-images.githubusercontent.com/24798364/43677871-43135a12-9801-11e8-9104-d675656cc617.png">


**Questions:**
N/A
